gui/wxpython/lmgr/frame.py

```diff
@@ -1862,7 +1862,7 @@ def OnShowAttributeTable(self, event, selection=None):
         # available only for vector map layers
         try:
             maptype = tree.GetLayerInfo(layer, key="maplayer").type
-        except:
+        except (AttributeError, KeyError):
```

Comment:
Looks similar to above, so why the keyError?

Reply:
I had removed the instances of `KeyError`, looks like I missed these 2 instances